pact/coin-contract/coin-faucet.pact

```diff
@@ -0,0 +1,94 @@
+(module coin-faucet FAUCET-GOVERNANCE
+
+  "'coin-faucet' represents Kadena's Coin Faucet Contract."
+
+  (defcap FAUCET-GOVERNANCE () true)
+
+  (use coin)
+
+  ; --------------------------------------------------------------------------
+  ; Schemas and Tables
+  ; --------------------------------------------------------------------------
+
+  (defschema history
+    @doc "Table to record of the behavior of addresses. Last transaction time,   \
+    \ total coins earned, and total coins returned are recorded per transaction. "
+    last-tx-time:time
+    total-coins-earned:decimal
+    total-coins-returned:decimal
+    )
+
+  (deftable history-table: {history})
+
+  ; --------------------------------------------------------------------------
+  ; Capabilities
+  ; --------------------------------------------------------------------------
+
+  (defcap HISTORY ()
```

Comment:
As far as I can tell this is not required. Table access is already guarded and HISTORY is only used to duplicate this. Generally, the only reason to use a module guard is to make some _external_ operation controlled by the module, ie the ownership of the faucet tokens in the coin contract. Inside of a module, the module operates on its tables with the capability inherently. The `defcap` should be removed.

Reply:
Removed HISTORY